elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreAliasDefinition.java

```diff
@@ -191,7 +191,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Res
             ServiceController<CredentialStore> serviceContainer = (ServiceController<CredentialStore>) context.getServiceRegistry(true).getRequiredService(credentialStoreServiceName);
             CredentialStore credentialStore = ((CredentialStoreService) serviceContainer.getService()).getValue();
             try {
-                if (entryType == null || ClearPassword.ALGORITHM_CLEAR.equals(entryType)) {
+                if (entryType == null || SUPPORTED_CREDENTIAL_TYPES[0].getCanonicalName().equals(entryType)) {
```

Comment:
I would properly fill "allowedValues" for ENTRY_TYPE attribute (also discard "Other"). 
At line 77 change getName to getCanonicalName.

 Then check:
if (entryType == null || entryType.equals(PasswordCredential.class.getCanonicalName())